### PR TITLE
feat: allow to pop a shell on Kubernetes

### LINKED ISSUE #732: make kurtosis service shell work on Kuberentes
### Background & motivation

Currently to do this I have to do a few things use kubectl

1. List namespaces
2. List pods in the namespace
3. Launch sh

This is a consuming workflow to get inside a container

After then I can debug

### Desired behaviour

kurtosis service shell enclave-name service-name

just works

### How important is this to you?

Painful; the lack of this feature makes using Kurtosis frictionful.

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
## Description:
Previously popping a shell was three different kubectl commands. Now its one Kurtosis command

## Is this change user facing?
YES

### CODE REVIEW

**Comment by h4ck3rk3y** [comment]
Uploading Screen Recording 2023-06-20 at 12.58.24 PM.mov…

**Comment by h4ck3rk3y** [comment]
https://github.com/kurtosis-tech/kurtosis/assets/6670312/f7771b97-c87e-4609-9aa9-eddf9deb440d

File: container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/kubernetes_kurtosis_backend.go [hunk 1]
```diff
@@ -316,22 +315,17 @@ func (backend *KubernetesKurtosisBackend) RunUserServiceExecCommands(
...
-		}
-	*/
-
-	// TODO IMPLEMENT
-	return nil, stacktrace.NewError("Getting a connection with a user service isn't yet implemented on Kubernetes")
+func (backend *KubernetesKurtosisBackend) GetShellOnUserService(ctx context.Context, enclaveUuid enclave.EnclaveUUID, serviceUuid service.ServiceUUID) (resultErr error) {
+	//namespaceName, err := shared_helpers.GetEnclaveNamespaceName(ctx, enclaveUuid, backend.cliModeArgs, backend.apiContainerModeArgs, backend.engineServerModeArgs, backend.kubernetesManager)
+	//if err != nil {
+	//	return nil, stacktrace.Propagate(err, "An error occurred getting namespace name for enclave '%v'", enclaveUuid)
+	//}
```
Comment (gbouv) [inline container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/kubernetes_kurtosis_backend.go]: Can we remove this commented part?

**Review by gbouv — APPROVED** [review]
As long as you tested it and it works 🤷

**Pull request merged**